ctru-rs/examples/camera-image.rs

```diff
@@ -0,0 +1,138 @@
+use ctru::console::Console;
+use ctru::gfx::{Screen, Side};
+use ctru::services::cam::{Cam, CamOutputFormat, CamShutterSoundType, CamSize, Camera};
+use ctru::services::hid::KeyPad;
+use ctru::services::{Apt, Hid};
+use ctru::Gfx;
+use std::time::Duration;
+
+const WIDTH: usize = 400;
+const HEIGHT: usize = 240;
+const BUF_SIZE: usize = WIDTH * HEIGHT * 2 * 2;
```

Comment:
As part of the documentation, please add a comment explaining why it's multiplied by 2 twice.